### PR TITLE
Fix queues_ignored parameter

### LINKED ISSUE #202: [RabbitMQ] Parameter queues_ignored invalid.
All ini parameters are strings so when we set:
```
ignored_queues = foo
```
then in collector we would have list ['f', 'o', 'o'] instead of ['foo'].

### PR DESCRIPTION
**Pull request by Nihn**
Fix #202.

### CODE REVIEW

File: src/collectors/rabbitmq/rabbitmq.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def collect(self):
         self.collect_health()
         matchers = []
         if self.config['queues_ignored']:
-                for reg in self.config['queues_ignored']:
+                for reg in self.config['queues_ignored'].split():
```
Comment (mattrobenolt) [inline src/collectors/rabbitmq/rabbitmq.py]: Do you want to use commas here instead of splitting on spaces? Do we use spaces elsewhere as a delimiter?

Comment (Nihn) [inline src/collectors/rabbitmq/rabbitmq.py]: Yes, spaces are used in [vhost] section like:
```
[vhost]
/ = queue1 queue2
```

Comment (mattrobenolt) [inline src/collectors/rabbitmq/rabbitmq.py]: That's gross, but ok. :)

**Comment by mattrobenolt** [comment]
Thanks. :cake:

**Pull request merged**